.travis.yml

```diff
@@ -86,11 +86,14 @@ install:
     - source continuous-integration/travis/openjpeg.sh  # so glymur knows where to look
 
 before_script:
+    # configure a headless display to test matplotlib
+    - "export DISPLAY=:99.0"
+    - "sh -e /etc/init.d/xvfb start"
```

Comment:
Thanks, I will try that. I was hoping to start a stab at reducing doc warnings where possible. I wanted to see if this suppressed the matplotlib warning.